app/sensors.go

```diff
@@ -162,6 +162,95 @@ func getEntities(w *wallbox.Wallbox) map[string]Entity {
 				"suggested_display_precision": "1",
 			},
 		},
+		"power_boost_power_l1": {
+			Component: "sensor",
+			Getter: func() string {
+				return fmt.Sprint(w.Data.RedisM2W.PowerBoostLine1Power)
+			},
+			Config: map[string]string{
+				"name":                        "Power Boost L1",
```

Comment:
Minor nit: I'd rename `Power Boost` to something more generic like `External Energy Meter` or similar.

Reply:
I agree that the name is not ideal, but it seems like Wallbox also supports [MID type energy meters](https://support.wallbox.com/na/knowledge-base/accessory/#power-meter), so I think anything else would be more confusing if somebody adds support for those. I don't like calling it power boost either, so I'm open to ideas though. :)